Python/Product/PythonTools/PythonToolsService.cs

```diff
@@ -661,52 +661,7 @@ public sealed class PythonToolsService : IDisposable {
         #endregion
 
         internal Dictionary<string, string> GetFullEnvironment(LaunchConfiguration config) {
```

Comment:
moved this code below into a new function which gets environmental variables by calling activate